core/scripts/cre/environment/configs/workflow-don.toml

```diff
@@ -35,6 +35,7 @@
 [extra_capabilities]
 # uncomment as needed and adjust paths to enable these capabilities and have them copied to containers/pods
 # these default paths assume that binaries are placed in core/scripts/cre/environment/ folder
+cron_capability_binary_path = "./cron"
```

Comment:
pls comment it back, I am working on a solution for local binaries so we don't have to go around and comment && uncomment them

Reply:
Sounds good